Source/Core/Core/IOS/Network/KD/Mail/WC24Send.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright 2023 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "Core/IOS/Network/KD/Mail/WC24Send.h"
+#include "Core/IOS/FS/FileSystem.h"
+#include "Core/IOS/Uids.h"
+
+namespace IOS::HLE::NWC24::Mail
+{
+constexpr const char SEND_LIST_PATH[] = "/" WII_WC24CONF_DIR "/mbox"
+                                        "/wc24send.ctl";
+
+WC24SendList::WC24SendList(std::shared_ptr<FS::FileSystem> fs) : m_fs{std::move(fs)}
+{
+  ReadSendList();
+}
+
+void WC24SendList::ReadSendList()
+{
+  const auto file = m_fs->OpenFile(PID_KD, PID_KD, SEND_LIST_PATH, FS::Mode::Read);
+  if (!file || !file->Read(&m_data, 1))
+    return;
```

Comment:
I assume the file not existing is normal operation (eg. on initial system menu boot), but if it exists but isn't large enough should there be a error log? We have a lot of users with questionable NANDs out there...

Reply:
There probably should, if the file is corrupted the Wii Menu will display the `System Files are corrupted` error on any attempt to launch the message board.